pwiz_tools/Skyline/Model/Irt/RCalcIrt.cs

```diff
@@ -243,6 +244,38 @@ public IEnumerable<DbIrtPeptide> GetDbIrtPeptides()
             IEnumerable<IRetentionTimeProvider> providers, int countProviders,
             DbIrtPeptide[] standardPeptideList, DbIrtPeptide[] items)
         {
+            var matchedStandard = IrtStandard.WhichStandard(standardPeptideList.Select(pep => pep.ModifiedTarget));
+            if (matchedStandard != null)
+            {
+                var dummyDoc = new SrmDocument(SrmSettingsList.GetDefault());
+                using (var reader = matchedStandard.DocumentReader)

```

Comment:
You should change the DocumentReader property into a function "GetDocumentReader()".

It's not a good idea to have a property where the caller is expected to call "Dispose" on the value.